src/api/routes/users.ts

```diff
@@ -247,6 +247,29 @@ const listPriviledgedUserDataAccess = procedure
   });
 });
 
+const remove = procedure
```

Comment:
"remove" => "delete" to be consistent with sql terms

Reply:
could not use word "delete" because it is a reserved word in Javascript. How about "deleteUser" ?

Reply:
ah. okay then remove is good